### DISCUSSION TITLE
[feature-req] AI integration

### DISCUSSION THREAD

**Discussion by craziedde** [espanso/espanso#1605] [Feature requests and Ideas]
Hi,

Generate AI response on selected text
(Optional feature: add user prompt for formatting proper question on selected text)

Thank you

**Comment by smeech** [comment]
Sorry this has hung here so long.

Would https://github.com/rohitna/chatgpt-script provide some of what you're looking for?

**Comment by craziedde** [comment]
> Would https://github.com/rohitna/chatgpt-script provide some of what you're looking for?

Thank you for your reply
Indeed, an interesting option for Linux users
Would be nice to have GUI version for windows

**Reply by smeech** [reply]
Ah - I didn't spend enough time reading the repo to spot that it's Linux- and MacOS-based. It would take a bit of work to modify it for Windows I guess (and the only public fork doesn't). A GUI version would be good.

**Comment by thoughtfuldata** [comment]
Any follow up on this?

**Reply by smeech** [reply]
Not as far as the Espanso package is concerned as far as I am aware, I'm afraid. Attention is focused on updating the main package currently. 
It may be worth keeping an eye on the above - someone might convert its `install.sh` for Windows, as the rest appears pretty platform independent.
